judgels-backends/jophiel/jophiel-app/src/main/java/judgels/jophiel/JophielModule.java

```diff
@@ -4,12 +4,22 @@
 import dagger.Provides;
 import javax.inject.Singleton;
 import judgels.jophiel.api.session.Session;
+import judgels.jophiel.session.SessionConfiguration;
 import judgels.jophiel.session.SessionStore;
 import judgels.service.actor.ActorChecker;
 
 @Module
 public class JophielModule {
-    private JophielModule() {}
+    private final JophielConfiguration config;
+
+    public JophielModule(JophielConfiguration config) {
```

Comment:
Instead of modifying this module, can we have `SessionModule` which provides the config?

This is the current pattern in Jophiel.